Libraries/Opc.Ua.Server/Configuration/ConfigurationNodeManager.cs

```diff
@@ -602,44 +631,69 @@ private ServiceResult CreateSigningRequest(
                 throw new ArgumentNullException(nameof(subjectName));
```

Comment:
At first glance shouldn't we throw ArgumentNullException if (String.IsNullOrEmpty(subjectName)) 
 not when non null or Enmpty?